src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -61,12 +63,12 @@ export default function SegmentedReqProvider(es, Private, Promise, timefilter, c
       // parameters via the handle
       if (_.isFunction(this._initFn)) this._initFn(this._handle);
       return this._createQueue().then((queue) => {
+        if (this.stopped) return;
+
         this._all = queue.slice(0);
 
         // Send the initial fetch status
         this._reportStatus();
-
-        return super.start();
```

Comment:
Yeah, I went looking for the discussion we had about it, but I'm pretty sure it's just what I talked about in the last paragraph of my description:

> It feels wrong to me that the request is marked "started" before it's start() method is done, but the only other solution I have in mind requires that only one courier fetch cycle can initialize at a time (a much more impactful change).